.github/workflows/retag_image.yml

```diff
@@ -53,17 +53,25 @@ jobs:
           [ $? -ne 0 ] && exit 1
 
           status=0
-          OUTPUT=$(aws ecr put-image --repository-name ${{ secrets.AWS_RESOURCE_NAME_PREFIX }} --image-tag ${{ inputs.image_tag }} \
+
+          #Split tag input
+          IFS=,
+          read -ra my_tags <<< "${{ inputs.image_tag }}"
+          
+          for tag in "${my_tags[@]}";
+          do
+            OUTPUT=$(aws ecr put-image --repository-name ${{ secrets.AWS_RESOURCE_NAME_PREFIX }} --image-tag $tag \
```

Comment:
I may be reading this incorrectly.  Is this going to run `aws ecr put-image` once per tag? I thought we want to run this command once (with one --image-tag per tag provided).